### PR TITLE
Tour through a button

### PR DESCRIPTION
**Pull request by FauDavid**
Add a quick start guide. The system must be capable of making an interactive tour (tour) guiding the user to take the first steps to activate the operation of the system.

### CODE REVIEW

File: src/components/notifications-preferences.component.js [hunk 1]
```diff
@@ -1,414 +1,443 @@
...
+                    active={activeIndex === 1}
+                  >
+                    <Grid celled="internally" stackable columns={2}>
+                      <Grid.Row>
+                        <Grid.Column>
+                          <Table
+                            className="animated fadeIn"
+                            unstackable
+                            basic="very"
+                            style={{ margin: "20px" }}
```
Comment (AgustinSpahn) [inline src/components/notifications-preferences.component.js:285]: Styles could be set in a css file

File: src/components/tour/steps-dashboard.component.js [hunk 2]
```diff
@@ -0,0 +1,612 @@
...
+      when: {
+        show: () => {
+          props.history.push("/dashboard/data_collectors");
+        },
+      },
+    },
+    {
+      id: "new_data_sources",
+      title: `<b>New Data Source<b>`,
+      text: "This button can allow you to open a form to add a new data source",
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: "This button allows you to open a form to add a new data source"

File: src/components/tour/steps-dashboard.component.js [hunk 3]
```diff
@@ -0,0 +1,612 @@
...
+      when: {
+        show: () => {
+          props.history.push("/dashboard/events_manager");
+        },
+      },
+    },
+    {
+      id: "events_manager",
+      title: `<b>Events Manager<b>`,
+      text: `This section can allow you to configure the notifications of the system.`,
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: `This section allows you to configure the system notifications.`

File: src/components/tour/steps-dashboard.component.js [hunk 4]
```diff
@@ -0,0 +1,612 @@
...
+      when: {
+        show: () => {
+          props.history.push("/dashboard/events_manager");
+        },
+      },
+    },
+    {
+      id: "sources",
+      title: `<b>Sources<b>`,
+      text: `These switches can allow you to select which data sources should generate notifications.`,
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: `These switches allow you to select which data sources should generate notifications.` (maybe "send notifications?")

File: src/components/tour/steps-dashboard.component.js [hunk 5]
```diff
@@ -0,0 +1,612 @@
...
+        },
+        hide: () => {
+          document.getElementById("triggers").click();
+        },
+      },
+    },
+    {
+      id: "action",
+      title: `<b>Action<b>`,
+      text: `These switches can allow you to configure the type of notifications to receive: push or email notifications.`,
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: remove "can"

File: src/components/tour/steps-dashboard.component.js [hunk 6]
```diff
@@ -0,0 +1,612 @@
...
+      when: {
+        show: () => {
+          props.history.push("/dashboard/resources_usage");
+        },
+      },
+    },
+    {
+      id: "network_overview",
+      title: `<b>Network Overview<b>`,
+      text: `This section displays statistics about the existing devices on the network. Also, you can use the filters or search bar. It shows graphs by status, flow messages and signal strength. The list can allow you to interact with the items on the list by clicking on one.`,
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: `[...] The list allows you to interact with its items by clicking on them.`

File: src/components/tour/steps-dashboard.component.js [hunk 7]
```diff
@@ -0,0 +1,612 @@
...
+      when: {
+        show: () => {
+          props.history.push("/dashboard/inventory");
+        },
+      },
+    },
+    {
+      id: "inventory",
+      title: `<b>Inventory<b>`,
+      text: `This section lists the existing devices on the network. Also, you can use the filters or search bar. It shows graphs by vendor, data source and importance.The list can allow you to interact with the items on the list by clicking on one or checking the box to set importance or assign labels.`,
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: same as last one

File: src/components/tour/steps-dashboard.component.js [hunk 8]
```diff
@@ -0,0 +1,612 @@
...
+      when: {
+        show: () => {
+          props.history.push("/dashboard/alerts_review");
+        },
+      },
+    },
+    {
+      id: "alerts",
+      title: `<b>Alerts<b>`,
+      text: `This section displays the existing alerts on the system. Also, you can use the filters or search bar. It shows graphs by risk, description and data source. The list can allow you to interact with the items on the list by clicking on one.`,
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: same as last one

File: src/components/tour/steps-dashboard.component.js [hunk 9]
```diff
@@ -0,0 +1,612 @@
...
+      when: {
+        show: () => {
+          props.history.push("/dashboard/current_issues");
+        },
+      },
+    },
+    {
+      id: "current_issues",
+      title: `<b>Current Issues<b>`,
+      text: `This section displays the current issues on the network. While an alert is a specific event in time that can affect the network, an issue is a problem/vulnerability that is persistent. Also, you can use the filters or search bar. It shows graphs by risk, description and data source. The list can allow you to interact with the items on the list by clicking on one.`,
```
Comment (AgustinSpahn) [inline src/components/tour/steps-dashboard.component.js]: same as last one

**Review by AgustinSpahn — CHANGES_REQUESTED** [review]
Looks good to me!

**Review by AgustinSpahn — DISMISSED** [review]

**Comment by AgustinSpahn** [comment]
Looks good to me!

**Review by cyones — APPROVED** [review]

**Pull request merged**